### PR TITLE
fix(aws-cloudfront): do not override aliases when not specified in serverless.yml inputs

### PR DESCRIPTION
**Pull request by dphang**
This fixes: https://github.com/serverless-nextjs/serverless-next.js/issues/703

Essentially, we should not be destructive when inputs are not specified for CloudFront. If aliases is set to `[]` then it will override with empty aliases, but if undefined it should not override existing aliases.

@dpowell and @LuisFros can you please help pull this change, build locally and test it since you have been using this.

### CODE REVIEW

**Comment by LuisFros** [comment]
@dphang I just ran the deploy locally and it works!

File: packages/serverless-components/aws-cloudfront/__tests__/general-options.test.js [hunk 1]
```diff
@@ -130,6 +130,25 @@ describe("General options propagation", () => {
...
   });
 
+  it("update distribution with undefined aliases does not override existing aliases", async () => {
+    // Create distribution
+    await component.default({ enabled: true, origins });
+
+    // Update distribution
+    await component.default({
+      enabled: false,
+      origins
```
Comment (LuisFros) [inline packages/serverless-components/aws-cloudfront/__tests__/general-options.test.js:140]: Complementing your test, you could do the following. 
```suggestion
      aliases: ["foo.example.com"],
      origins
```

**Comment by dphang** [comment]
@LuisFros thanks for confirming

**Comment by dpowell** [comment]
Works well here, too. Thanks!

**Pull request merged**